### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by bm371613**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+In this repository we open-source a method for training interpretable models,
+originally developed for ransomware attack detection.
+
+Key features:
+- model and prediction interpretability
+- concise implementation based on `numpy` and `pytorch`
+- efficiency: training can be executed on a CPU, evaluation is fast
+- extensibility: model structure allows easy enforcement of task-specific requirements by additional regularization
+
+We also include everything necessary to reproduce experiments we conducted to publish our results.
```
Comment (mayagos) [inline README.md]: the experiments

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+
+## How it works
+
+All the implementation is in [lib/embedding_sum.py](lib/embedding_sum.py) and consists of three classes:
```
Comment (mayagos) [inline README.md]: The entire implementation can be found in

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+ - Digitizer
+ - EmbeddingSumModule
+ - EmbeddingSumClassifier
+
+The method can be summarized as follows:
+
+1. For each feature, the `Digitizer` defines quantile-based bins and encodes feature values as bin ordinals,
+2. The `EmbeddingSumModule` defines a parameter for each bin of each feature and implements evaluation as sum of the
+   parameters corresponding to bin ordinals in the input vector,
```
Comment (mayagos) [inline README.md]: The word parameter is very vague. Is there another term that doesn't go against the literature that you could use? Something more descriptive.

File: README.md (hunk 1 above)
Comment (bm371613) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (bm371613) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (bm371613) [reply README.md]: now: trainable parameter

**Pull request merged**